ext/intl/dateformat/dateformat_parse.c

```diff
@@ -145,18 +154,59 @@ PHP_FUNCTION(datefmt_parse)
 			RETURN_FALSE;
 		}
 		parse_pos = (int32_t)long_parse_pos;
-		if((size_t)parse_pos > text_len) {
+		if ((size_t)parse_pos > text_len) {
 			RETURN_FALSE;
 		}
 	}
-	internal_parse_to_timestamp( dfo, text_to_parse, text_len, z_parse_pos?&parse_pos:NULL, return_value);
-	if(z_parse_pos) {
+	internal_parse_to_timestamp( dfo, text_to_parse, text_len, z_parse_pos ? &parse_pos : NULL, false, return_value);
+	if (z_parse_pos) {
 		zval_ptr_dtor(z_parse_pos);
 		ZVAL_LONG(z_parse_pos, parse_pos);
 	}
 }
 /* }}} */
 
+PHP_METHOD(IntlDateFormatter, parseToCalendar)
+{
+	zend_string    *text_to_parse = NULL;
+	zval*         	z_parse_pos = NULL;
+	int32_t		parse_pos = -1;
```

Comment:
I wouldn't bother aligning it
```suggestion
	zend_string *text_to_parse = NULL;
	zval* z_parse_pos = NULL;
	int32_t parse_pos = -1;
```